diff-models/core/__tests__/gen.test.ts

```diff
@@ -0,0 +1,12 @@
+import { gen } from '../src/main.js';
```

Comment:
Is the stuff in this folder necessary?

Reply:
The easiest way I know of to check the statement coverage of the trace generation on the model is to use `yarn jest --collect-coverage` which requires a jest test.
I don't think we should give this up